### DISCUSSION TITLE
Problems with systemctl --user

### DISCUSSION THREAD

**Discussion by baumrasen** [photobooth-app/photobooth-app#278] [General]
I had problems with the "systemctl --user"-commands from the installation guide. I always get the error:

> failed to connect to bus: no medium found

I made the mistake to run the command in a ssh-console. There this command is not working. **You have to use the console on the desktop.** Then all is working as expected. Maybe you can add a hint in the dokumentation for other people. 

I'm on a raspberry pi 4 with fresh debian bookworm installation.

**Comment by mgineer85** [comment]
That’s strange, I only use ssh. Maybe a very recent change?

**Comment by baumrasen** [comment]
Okay. Thanks for your answer. I checked this. And it works with ssh.

The reason was, that I use tmux with ssh and normaly start tmux with sudo. Then in the tmux session I use
> su - username
to run a tmux window with username-bash
There the "systemctl --user ..." commands does not work.

So the cause is my "special setup" - and not the latest bookworm image!
